packages/payments-plugin/e2e/paypal-dev-server.ts

```diff
@@ -0,0 +1,118 @@
+import { AdminUiPlugin } from '@vendure/admin-ui-plugin';
+import { DefaultLogger, DefaultSearchPlugin, LanguageCode, LogLevel, mergeConfig } from '@vendure/core';
+import { createTestEnvironment, registerInitializer, SqljsInitializer, testConfig } from '@vendure/testing';
+import gql from 'graphql-tag';
+import path from 'path';
+
+import { initialData } from '../../../e2e-common/e2e-initial-data';
+import { PayPalPlugin } from '../src/paypal/paypal.plugin';
+
+import { CREATE_PAYMENT_METHOD } from './graphql/admin-queries';
+import {
+    CreatePaymentMethodMutation,
+    CreatePaymentMethodMutationVariables,
+} from './graphql/generated-admin-types';
+import { ADD_ITEM_TO_ORDER, APPLY_COUPON_CODE, TRANSITION_TO_STATE } from './graphql/shop-queries';
+import { setShipping } from './payment-helpers';
+import { PayPalSdkPlugin } from './paypal-sdk.plugin';
+
+/**
+ * This should only be used to locally test the PayPal payment plugin
+ * Make sure you have `PAYPAL_CLIENT_ID=xxxx`, `PAYPAL_CLIENT_SECRET=xxxx`
+ * and `PAYPAL_MERCHANT_ID=xxxx` in your .env file
+ */
+/* eslint-disable @typescript-eslint/no-floating-promises */
+async function runPayPalDevServer() {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    require('dotenv').config();
+
+    const { PAYPAL_CLIENT_ID, PAYPAL_CLIENT_SECRET, PAYPAL_MERCHANT_ID } = process.env;
+    if (!PAYPAL_CLIENT_ID || !PAYPAL_CLIENT_SECRET || !PAYPAL_MERCHANT_ID) {
+        throw new Error('Please provide PayPal credentials in the .env file');
+    }
+
+    const apiUrl = 'https://api-m.sandbox.paypal.com';
```

Comment:
I know its probably unlikely for paypal to change their sandbox URL but it could prove useful to get this URL also from the .env with the current value as default.